src/com/company/redi_exercises/Main.java

```diff
@@ -0,0 +1,186 @@
+package com.company.redi_exercises;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        //fifthExercise(scanner);
+        //seventhExercise(scanner);
+        //eighthExercise();
+        //twelfthExercise();
+        //thirteenthExercise();
+        //thirteenthExercisePartTwo(scanner);
+        //fourteenthExercise();
+
+    }
+
+    private static void fifthExercise(Scanner scanner) {
+        int numberWords = scanner.nextInt();
+        ArrayList<String> inputWords = new ArrayList<>();
+        for (int i = 0; i <= numberWords; i++) {
+            inputWords.add(scanner.nextLine());
+        }
+        System.out.println("Min length: ");
+        int numberLetters = scanner.nextInt();
+        System.out.println(removeShortWords(inputWords, numberLetters));
+    }
+
+    public static ArrayList<String> removeShortWords(ArrayList<String> words, int minLength) {
+        ArrayList<String> earnedWords = new ArrayList<>();
+        for (String word : words) {
+            if (word.length() > minLength || word.length() == minLength) {
```

Comment:
could you simplify it, I mean get rid of OR operator? how we check if value is more or equals to?

Reply:
fixed